Cherry-pick: "Added Tip to Variables for using Custom Company Fields" to 5.2 (#628)
mautic/user-documentation: docs/configuration/variables.rst

   * - Zip Code (Company)
     - ``{contactfield=companyzipcode}``

.. tip::
  
   **Custom Company fields**

>    The syntax for custom Company fields differs from core Company field syntax. You must **not** add the word "Company" in the variable and instead treat it as a ``contactfield``.

Reviewer: ```suggestion
   The syntax for custom Company fields differs from core Company field syntax. You must **not** add the word 'Company' in the variable and instead treat it as a ``contactfield``.
```

>    The syntax for custom Company fields is different than the core Company field syntax. You must **not** add the word "Company" in the variable; instead, treat it as a ``contactfield``.

Author: ```suggestion
   The syntax for custom Company fields differs from core Company field syntax. You must **not** add the word "Company" in the variable and instead treat it as a ``contactfield``.
```
